src/client/src/app/components/player-events/player-event-details/player-event-details.component.html

```diff
@@ -36,6 +36,16 @@
           </div>
         </p-card>
 
+        @if (externalUri() && externalUri() !== null && externalUri()?.length !== 0) {
```

Comment:
A string is truthy when it is not null, undefined or empty.

```suggestion
        @if (externalUri()) {
```